### PR TITLE
Add  env variabels to readme

### PR DESCRIPTION
**Pull request by ddonahue007**
[AAP-6875](https://issues.redhat.com/browse/AAP-6875)

 readme for local dev setup

Need to set the following variable when running in the dev env(using the `dev` tasks)

`export EDA_DEPLOYMENT_TYPE=local`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,14 +58,20 @@ Install Ansible and `ansible.eda` collection:
 pip install ansible
 ansible-galaxy collection install ansible.eda
 ```
+### 3. Set up env variables
+  Set up the following env variable(s) when testing/running local dev env
 
-### 3. Services
+```shell
+export EDA_DEPLOYMENT_TYPE=local
```
Comment (Alex-Izquierdo) [inline README.md:65]: Why? I don't see this envvar used anywhere

Comment (ddonahue007) [reply README.md:65]: The default is set here: 
https://github.com/ansible/eda-server/blob/main/src/eda_server/config/__init__.py#L23

**Review by hsong-rh — APPROVED** [review]
👍

File: README.md (hunk 1 above)
Comment (ddonahue007) [reply README.md:65]: I am going tp move this to a note and reference that this is only needed as a temporary setting.

**Review by Alex-Izquierdo — APPROVED** [review]

**Pull request merged**